test/unittest/CMakeLists.txt

```diff
@@ -1,22 +1,60 @@
 # ---[ Google Test
+if(MSVC)
+  if(MSVC_VERSION LESS 1900)
+    message(FATAL_ERROR "Need Visual Studio 2015 or newer to compile unit tests")
+  endif()
+endif()
+
 if (UNIX)
-    SET(CMAKE_EXE_LINKER_FLAGS "-pthread")
+  SET(CMAKE_EXE_LINKER_FLAGS "-pthread")
 endif(UNIX)
+
 enable_testing()
 find_package(Threads REQUIRED)
+
 file(GLOB_RECURSE UNIT_TEST_SOURCE "*.cc")
 add_executable(${PROJECT_NAME}_unit_tests ${UNIT_TEST_SOURCE})
 set_property(TARGET ${PROJECT_NAME}_unit_tests
-        PROPERTY RUNTIME_OUTPUT_DIRECTORY ${PRIVATE_RUNTIME_DIR})
+  PROPERTY RUNTIME_OUTPUT_DIRECTORY ${PRIVATE_RUNTIME_DIR})
 
-message(STATUS "${CMAKE_CURRENT_SOURCE_DIR}/build_config.h.in -> ${CMAKE_CURRENT_SOURCE_DIR}/build_config.h")
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/build_config.h.in" "${CMAKE_CURRENT_SOURCE_DIR}/build_config.h")
+message(STATUS "${CMAKE_CURRENT_SOURCE_DIR}/build_config.h.in -> ${CMAKE_CURRENT_BINARY_DIR}/build_config.h")
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/build_config.h.in" "${CMAKE_CURRENT_BINARY_DIR}/build_config.h")
+target_include_directories(${PROJECT_NAME}_unit_tests PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Isn't this duplicate of line 25?

Reply:
Ops, something went wrong during rebase.  Thanks.